content/en/continuous_integration/setup_tests/python.md

```diff
@@ -39,36 +32,53 @@ pip install "ddtrace>=0.50.0rc4"
 
 For more information, see the [Python tracer installation documentation][2].
 
-## Instrumenting your pytest tests
+## Instrumenting your tests
 
-To enable instrumentation of `pytest` tests, add the `--ddtrace` option when running `pytest`:
+To enable instrumentation of `pytest` tests, add the `--ddtrace` option when running `pytest`, specifiying the name of the service or library under test in the `DD_SERVICE` environment variable, and the environment where tests are being run (i.e. `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable:
 
 {{< code-block lang="bash" >}}
-pytest --ddtrace
+DD_SERVICE=my-python-app DD_ENV=ci pytest --ddtrace
 {{< /code-block >}}
 
-You can also add the following configuration to any file used to configure `pytest` (such as `pytest.ini` or `setup.cfg`):
+## Additional configuration settings
 
-{{< code-block lang="ini" >}}
-[pytest]
-ddtrace = 1
-{{< /code-block >}}
+The following is a list of the most important configuration settings that can be used with the tracer, either in code or using environment variables:
+
+`ddtrace.config.service`
+: Name of the service or library under test.<br/>
+**Environment variable**: `DD_SERVICE`<br/>
+**Default**: `pytest`<br/>
+**Example**: `my-python-app`
+
+`ddtrace.config.env`
+: Name of the environment where tests are being run.<br/>
+**Environment variable**: `DD_ENV`<br/>
+**Default**: `(empty)`<br/>
+**Examples**: `local`, `ci`
+
+The following configuration settings can be passed in as parameters to `tracer.configure()`, or using environment variables:
+
+`enabled`
+: Setting this to `false` completely disables the instrumentation.<br/>
+**Environment variable**: `DD_TRACE_ENABLED`<br/>
+**Default**: `true`
```

Comment:
This seems like a weird option. Does removing `--ddtrace` do the same thing?

Reply:
Yes, in the case of Python is weird, and I don't think it's worth having it for consistency. I'll remove it.